context-menu/vertical-context-navigation.css

```diff
@@ -0,0 +1,92 @@
+/*
+ * Make the context-menu navigation items vertical
+ *
+ * Contributor(s): MrOtherGuy
+ */
+
+/* Set the container vertical and disable the its pointer events */
+#contentAreaContextMenu #context-navigation:not([hidden]){ 
+  -moz-box-orient: vertical !important;
+  position:absolute !important;
+  background-color: transparent !important;
+  pointer-events: none;
+}
+
+/* Enable individual buttons' pointer-events which would otherwise be disabled through cascade */
+#contentAreaContextMenu #context-navigation:not([hidden]) menuitem{
+  pointer-events: auto !important;
+  position:absolute !important;
+  margin-left:0px;
+}
+
+/* Move all context-menu entries right except the navigation items */ 
+#contentAreaContextMenu #context-navigation:not([hidden])~*{
+  margin-left: 30px !important;
```

Comment:
Moving this to `60px` fixes the overlap. Only a margin above the top menu left :)

Reply:
Oh, and there's still a weird hover effect that hides part of the other menu items...

Reply:
Is the weird hover effect like as if the context navigation box was as wide as the whole menu - covering all topmost items? That would be explained because in reality it is exactly that. It's just supposed to be totally transparent except for the buttons themselves.

Honestly though, I don't think I can do much about this before I manage to set up a linux box to test things. Probably in the next few days.

Reply:
No. It looks like this now: https://i.imgur.com/dHBwApX.png

Reply:
I have no idea what's going on there. 

What I do know though is that the icon margin fix didn't do anything (as the navigation button spacing is the same as before) and that is why there is still that empty space above menuitems - for the same reason the last element is cut in half. This is also something that I'd need to try out to understand why.

Reply:
Thank you for your effort! Let me know if you find something :)